time stamp

# time since previous message (as measured by the encoders)
float64 dt

# rotation on each wheel (number of revolutions)
float64 d_left
float64 d_right